src/backend/TrafficCourts/Citizen.Service/Validators/FormRecognizerValidator.cs

```diff
@@ -73,12 +76,106 @@ public static void Sanitize(OcrViolationTicket violationTicket)
         {
             string count3Section = violationTicket.Fields[OcrViolationTicket.Count3Section].Value ?? "";
             if (!violationTicket.Fields[OcrViolationTicket.Count3ActRegs].IsPopulated()
-                && count3Section.StartsWith("MVA"))
+                && count3Section.StartsWith("M") && count3Section.Length >= 3)
             {
-                violationTicket.Fields[OcrViolationTicket.Count3ActRegs].Value = "MVA";
-                violationTicket.Fields[OcrViolationTicket.Count3Section].Value = count3Section.Replace("MVA", "");
+                string actReg = count3Section.Substring(1, 3);
+                violationTicket.Fields[OcrViolationTicket.Count3ActRegs].Value = actReg;
+                violationTicket.Fields[OcrViolationTicket.Count1Section].Value = count3Section.Replace((actReg is not null ? actReg : ""), "")?.Trim();
             }
         }
+
+        // If any act regs starts with MU replace with MV since the U is likely a mis-scan of the V
+        if (violationTicket.Fields.ContainsKey(OcrViolationTicket.Count1ActRegs))
+            violationTicket.Fields[OcrViolationTicket.Count1ActRegs].Value = violationTicket.Fields[OcrViolationTicket.Count1ActRegs].Value?.Replace("MU", "MV");
+        if (violationTicket.Fields.ContainsKey(OcrViolationTicket.Count2ActRegs))
+            violationTicket.Fields[OcrViolationTicket.Count2ActRegs].Value = violationTicket.Fields[OcrViolationTicket.Count2ActRegs].Value?.Replace("MU", "MV");
+        if (violationTicket.Fields.ContainsKey(OcrViolationTicket.Count3ActRegs))
+            violationTicket.Fields[OcrViolationTicket.Count3ActRegs].Value = violationTicket.Fields[OcrViolationTicket.Count3ActRegs].Value?.Replace("MU", "MV");
+
+        // Pre-process ticket number if scan reads an A Oh as a A Zero replace the Zero with an O in the second position
+        // replace A1 with AI
+        if (violationTicket.Fields.ContainsKey(OcrViolationTicket.ViolationTicketNumber))
+        {
+            string? ticketNumber = violationTicket.Fields[OcrViolationTicket.ViolationTicketNumber].Value;
+            if (ticketNumber is not null && ticketNumber.StartsWith("A0"))
+            {
+                violationTicket.Fields[OcrViolationTicket.ViolationTicketNumber].Value = ticketNumber.Replace("A0", "AO");
+            }
+            if (ticketNumber is not null && ticketNumber.StartsWith("A1"))
+            {
+                violationTicket.Fields[OcrViolationTicket.ViolationTicketNumber].Value = ticketNumber.Replace("A1", "AI");
+            }
+        }
+
+        // Pre-process description and act/regs fields for counts if description number contains text of MVA or MVR strip it out and replace blank act/regs field text
+        if (violationTicket.Fields.ContainsKey(OcrViolationTicket.Count1ActRegs) && violationTicket.Fields.ContainsKey(OcrViolationTicket.Count1Description))

```

Comment:
The same can be done with these blocks. They all seem the same except for the two field names, 

```
        // Pre-process description and act/regs fields for counts if description number contains text of MVA or MVR strip it out and replace blank act/regs field text
        void ProcessActRegs(string regs, string description)
        {
            if (violationTicket.Fields.ContainsKey(regs) && violationTicket.Fields.ContainsKey(description))
            {
                string? desc = violationTicket.Fields[description].Value;
                bool actRegPop = violationTicket.Fields[regs].IsPopulated();
                if (desc is not null && (desc.Contains("MVA") || desc.Contains("MVR")) && actRegPop == false)
                {
                    string actReg = "MVA";
                    if (desc.Contains("MVR")) actReg = "MVR";
                    violationTicket.Fields[description].Value = violationTicket.Fields[description].Value?.Replace(actReg, "");
                    violationTicket.Fields[regs].Value = actReg;
                }
            }
        }

        ProcessActRegs(OcrViolationTicket.Count1ActRegs, OcrViolationTicket.Count1Description);
        ProcessActRegs(OcrViolationTicket.Count2ActRegs, OcrViolationTicket.Count2Description);
        ProcessActRegs(OcrViolationTicket.Count3ActRegs, OcrViolationTicket.Count3Description);
```